### PR TITLE
feat: use DAO service fee on the frontend

### PR DESCRIPTION
**Pull request by reisr3**

### CODE REVIEW

File: frontend/src/components/create-contract/PaymentSummary.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { DAO_SERVICE_FEE } from '../../constants';
```
Comment (deweller) [inline frontend/src/components/create-contract/PaymentSummary.tsx:1]: The Dao Fee can change over time.  We can pull this from the `DAO_FEE` method on the smart contract.

```typescript
  const [daoFee, daeFeeLoading] = useSmartContractCall(
    contracts.Job.DAO_FEE
  );
  const daoFeeAsNumber = daoFee?.toNumber() || 0;
```

**Review by deweller — CHANGES_REQUESTED** [review]
This looks good with one request.

Let's pull the DAO_FEE from the contract.  That way if it changes in the contract, we don't need to redeploy our front-end app.

File: frontend/src/components/create-contract/PaymentSummary.tsx (hunk 1 above)
Comment (reisr3) [reply frontend/src/components/create-contract/PaymentSummary.tsx]: It took me forever to add this gif, and you're right, done

**Review by deweller — APPROVED** [review]
Looks great

**Pull request merged**